.travis/linux..install.sh

```diff
@@ -15,3 +15,9 @@ else
 fi
 
 sudo apt-get install -y $PACKAGES
+
+# Fix header
```

Comment:
Non-Travis builds may not have sudo permission. We could add a task to warn about it and maybe fix it, but this is an issue that libgig upstream has dealt with and distros should handle their versions.